File: android/lib/src/main/java/duck/cameras/android/model/SettingsUrl.java
package duck.cameras.android.model;

public class SettingsUrl {
    private String value;

    public SettingsUrl(String value) {
        this.value = value;
    }

    public String value() {
        return value;
    }

    public boolean isEmpty() {
        return value == null || value.isEmpty();
    }
}

File: windows/Model/SettingsUrl.cs
namespace Duck.Cameras.Windows.Model
{
    public class SettingsUrl
    {
        public SettingsUrl(string value)
        {
            Value = value;
        }

        public string Value { get; private set; }

        public bool IsEmpty()
        {
            return string.IsNullOrEmpty(Value);
        }
    }
}
